llvm/lib/Target/RISCV/RISCV.td

```diff
@@ -369,6 +369,15 @@ def FeatureRV32E
 def IsRV32E : Predicate<"Subtarget->isRV32E()">,
                         AssemblerPredicate<(all_of FeatureRV32E)>;
 
+// TODO: Once all supported CheriBSD branches are ready for the new semantics
+// this feature should be enabled automatically.
+def FeatureNoCheriV8Compat
+    : SubtargetFeature<"xcheri-no-v8-compat", "HasCheriISAv8Semantics", "false",
+                       "CHERI ISAv8 semantics (trapping, DDC/PCC relocation)">;
```

Comment:
I originally started with a positive V9 semantics flag, but that did not out for some reason I've now forgotten since it's been a while. I *think* it caused problems when I tried to introduce the xcherimin feature.

Reply:
Should be better now?